src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -102,6 +102,13 @@ private static Task ExecuteObjectResult(object? obj, HttpContext httpContext)
             }
         }
 
+        private static Func<HttpContext, StringValues> ResolveFromRouteOrQuery(string parameterName, IEnumerable<string>? routeParameterNames)
+        {
+            return routeParameterNames?.Contains(parameterName, StringComparer.OrdinalIgnoreCase) == true
+                ? (httpContext) => new StringValues(httpContext.Request.RouteValues[parameterName]?.ToString())
```

Comment:
Nit: I know this isn't new, but doesn't RDF cast instead of call ToString()?

```suggestion
                ? (httpContext) => new StringValues((string?)httpContext.Request.RouteValues[parameterName]))
```